File: sub-model/src/main/java/org/github/base/service/impl/UserRoleServiceImpl.java
package org.github.base.service.impl;

import org.github.base.entity.UserRoleEntity;
import org.github.base.mapper.IUserRoleMapper;
import org.github.base.service.IUserRoleService;
import org.github.base.ServiceImpl;
import org.springframework.stereotype.Service;

/**
 * <p>
 *  服务实现类
 * </p>
 *
 * @author JYD_XL
 * @since 2021-04-21
 */
@Service
public class UserRoleServiceImpl extends ServiceImpl<IUserRoleMapper, UserRoleEntity> implements IUserRoleService {}

File: sub-mysql-blog-service/src/main/java/org/github/mysql/blog/base/service/impl/UserRoleServiceImpl.kt
@file:Suppress("RedundantSemicolon")

package org.github.mysql.blog.base.service.impl;

import org.github.mysql.blog.base.entity.UserRoleEntity;
import org.github.mysql.blog.base.mapper.IUserRoleMapper;
import org.github.mysql.blog.base.service.IUserRoleService;
import org.github.base.ServiceImpl;
import org.springframework.stereotype.Service;

/**
 *  服务实现类
 *
 * @author JYD_XL
 */
@Service
open class UserRoleServiceImpl : ServiceImpl<IUserRoleMapper, UserRoleEntity>(), IUserRoleService
